presentation/src/main/java/com/knocklock/presentation/lockscreen/LockNotiItem.kt

```diff
@@ -0,0 +1,135 @@
+package com.knocklock.presentation.lockscreen
+
+import android.graphics.Bitmap
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.knocklock.presentation.ui.theme.KnockLockTheme
+
+/**
+ * @Created by 김현국 2022/12/02
+ * @Time 3:06 PM
+ */
+
+data class Notification(
+    val icon: Bitmap?,
+    val appTitle: String,
+    val notiTime: String,
+    val title: String,
+    val content: String
+) {
+    companion object {
+        val Test = Notification(
+            icon = null,
+            appTitle = "Kakao",
+            notiTime = "Now",
+            title = "KnockLock app 개발중",
+            content = "이거 너무나 재미가 있는걸~"
+        )
+    }
+}
+
+@Composable
+fun LockNotiItem(
+    modifier: Modifier = Modifier,
+    notification: Notification = Notification.Test
```

Comment:
```suggestion
    notification: Notification
```

테스트 객체는 Prev에서 직접 넣게 구현하면 좋을 것 같아요! 👍

Reply:
@jaeryo2357 !! 그게 더 좋을것 같아요 반영할께용

Reply:
@jaeryo2357 추후에 notification에서 내용들이 null로 올 수 있는 가능성도 있기 때문에, app Intent를 가져오면서 그 경우에 여기도 반영해놓을께용